scripts/dev_setup.sh

```diff
@@ -346,7 +345,7 @@ function install_boogie {
   if [[ "$("$HOME"/.dotnet/dotnet tool list -g)" =~ .*boogie.*${BOOGIE_VERSION}.* ]]; then
     echo "Boogie $BOOGIE_VERSION already installed"
   else
-    "$HOME/.dotnet/dotnet" tool install --global Boogie --version "$BOOGIE_VERSION"
+    "$HOME/.dotnet/dotnet" tool update --global Boogie --version $BOOGIE_VERSION
```

Comment:
I intentionally changed this to install because update does it everytime even if the right version is already installed (which is checked above). Any reason you changed it back? I was assuming the check above deals with avoiding failure of install.

Reply:
install wasn't working.  When I updated BOOGIE_VERSION to 2.7.32 and ran dev_setup.sh, it didn't update the version.  In  other words, it executes the install command but does not update saying that boogie is already installed.

Reply:
Strange. It does work in CI and all different images we test (those are on a clean machine). Anyway, fine with me to change this.